src/Compilers/VisualBasic/Test/Semantic/Binding/Binder_Statements_Tests.vb

```diff
@@ -3741,6 +3741,54 @@ Done
 
         End Sub
 
+        <WorkItem(45158, "https://github.com/dotnet/roslyn/issues/45158")>
+        <Fact>
+        Public Sub EndWithSingleLineIf()
```

Comment:
Consider have a version of this test in library output that verifies the error case.